### DISCUSSION TITLE
Email Support

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#156] [Ideas]
While most people have email, having a general set of boxes for events was useful. I would propose some guidance here for organizers, as well as some generic support in a framework or organization to allow email for events.

[a comment by an automated account is not shown]

**Comment by rabryst** [comment]
One org I work with creates temporary mailing lists for the length of the event (plus some time either side for padding). Mailing list software is mostly free, and we can assign event organizers as admins. Is that the kind of thing you had in mind?

[a comment by an automated account is not shown]

**Comment by way0utwest** [comment]
Yes. For many organizers, setting things up is difficult or cumbersome. While a few don't have any issues, lots want a turnkey solution, so the more we can provide in either detailed instructions or services, the more likely we are to grow the number of organizers.

[a comment by an automated account is not shown]
